src/Illuminate/Database/Query/Builder.php

```diff
@@ -185,6 +185,7 @@ class Builder
         'rlike', 'regexp', 'not regexp',
         '~', '~*', '!~', '!~*', 'similar to',
         'not similar to', 'not ilike', '~~*', '!~~*',
+        'between',
```

Comment:
Why is this needed?

Reply:
It was for my initial implementation, I'm removing it right now.